Payload/bleemsync/etc/bleemsync/FUNC/0050_bleemsync.funcs

```diff
@@ -67,56 +70,12 @@ execute_bleemsync_func(){
 #-----------------------------------------------------------------------------#
   start=$(date +%s%N)
 
-  #Run on console bleemsync (This will be replaced in 1.0)
-  if [ -f "$mountpoint/system/bleemsync/BleemSync" ]
-  then
-    cd "$mountpoint/system/bleemsync"
-    chmod +x "BleemSync"
-    ./BleemSync
-    cd "/home/root"
-  fi
-
-  end=$(date +%s%N)
-  echo "[BLEEMSYNC](PROFILE) BleemSync DB processing took: $(((end-start)/1000000))ms to execute"
-#-----------------------------------------------------------------------------#
-  start=$(date +%s%N)
-
-  # Build external directory structure
-  mkdir -p "$mountpoint/system/bios"
-  mkdir -p "$mountpoint/system/preferences/system"
-  mkdir -p "$mountpoint/system/preferences/user"
-  mkdir -p "$mountpoint/system/preferences/autodimmer"
-  mkdir -p "$mountpoint/system/databases"
-  mkdir -p "$mountpoint/system/region"
-  mkdir -p "$mountpoint/system/ui"
-
-  # Copy the bios files
-  [ ! -f "$mountpoint/system/bios/romw.bin" ] && cp -fr "/gaadata/system/bios/"* "$mountpoint/system/bios"
-  # Copy the regional.pre
-  [ ! -f "$mountpoint/system/preferences/system/regional.pre" ] && cp -f "/gaadata/preferences/"* "$mountpoint/system/preferences/system"
-  # Copy out the user.pre
-  [ ! -f "$mountpoint/system/preferences/user/user.pre" ] && cp -f "/data/AppData/sony/ui/"* "$mountpoint/system/preferences/user"
-  # Copy out the auto dimming config
-  [ ! -f "$mountpoint/system/preferences/autodimmer/config.cnf" ] && cp -f "/data/AppData/sony/auto_dimmer/"* "$mountpoint/system/preferences/autodimmer"
-  # Copy out the region info
-  [ ! -f "$mountpoint/system/region/REGION" ] && cp -f "/gaadata/geninfo/"* "$mountpoint/system/region"
-  # Copy ui error log
-  [ ! -f "$mountpoint/system/ui/error.log" ] && cp -f "/data/sony/ui/"* "$mountpoint/system/ui"
-  sync
-
-  end=$(date +%s%N)
-  echo "[BLEEMSYNC](PROFILE) BleemSync data processing took: $(((end-start)/1000000))ms to execute"
-#-----------------------------------------------------------------------------#
-  start=$(date +%s%N)
-
   # Unmount partitons and create tmpfs - Shut system down on failure
   MOUNT_FAIL=0
-  umount /data || MOUNT_FAIL=1
-  umount /gaadata || MOUNT_FAIL=1
   # Create gaadata and data folders in tmp then mount over original folders
-  mkdir -p "/var/volatile/gaadatatmp" "/var/volatile/datatmp"
-  mount -o bind "/var/volatile/gaadatatmp" "/gaadata" || MOUNT_FAIL=1
-  mount -o bind "/var/volatile/datatmp" "/data" || MOUNT_FAIL=1
+  VOL_GAADATA="/var/volatile/gaadatatmp"
```

Comment:
Variables > hardcoded string that you have to search and replace if you decide to change it.